scripts/install.sh

```diff
@@ -138,7 +140,7 @@ make
 IFACE=$(route | grep '^default' | grep -o '[^ ]*$')
 FILENAME=`cat /sys/class/net/$IFACE/address`
 echo Generating Keys - It may take a while...
-./priv/c_dist/tpm_keygen > $INSTALL_DIR/$FILENAME 
+./priv/c_dist/tpm_keygen > $INSTALL_DIR/$FILENAME
```

Comment:
Ass the INSTALL_DIR finishes with a "/" this will create a path with double "/" no ?